src/BuildScriptGenerator/BuildScriptGenerator.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
I'm confused why this was the fix, since we're updating it to 2.2, but 2.1 is .NET Core LTS. Is it possible that the issue is the 2.1 patch version in the build machines that is the issue, and not the fact that it is 2.1?